premerge/main.tf

```diff
@@ -61,6 +61,14 @@ moved {
   to   = module.premerge_cluster1
 }
 
+module "premerge_cluster2" {
```

Comment:
does it make sense to name them for regions instead of numbers? like "premerge_cluster_west?"

Reply:
That's a better naming scheme than just numbers. I'll update it.